Sources/MapboxCoreNavigation/NavigationEventsManager.swift

```diff
@@ -361,7 +361,23 @@ open class NavigationEventsManager {
      - parameter description: A custom string used to describe the problem in detail.
      */
     public func sendFeedback(_ feedback: FeedbackEvent, type: FeedbackType, description: String? = nil) {
-        feedback.update(type: type, source: .user, description: description)
+        feedback.update(type: type.description, subtype: type.description, description: description)
```

Comment:
You set both `type` and `subtype` to be the same? Here and in the next method too.

Reply:
Yeah, this was a typo, thanks for noticing.